src/main/kotlin/step3/view/ResultView.kt

```diff
@@ -0,0 +1,23 @@
+package step3.view
+
+import step3.Car
+
+object ResultView {
+    fun output(cars: ArrayList<Car>) {
+        print("output for turn\n")
+        for (i in 0 until cars.size) {
+            print(visualize(cars[i].distance))
+        }
+    }
+
+    fun visualize(distance: Int): String {
+        val block = "-"
+        var blocks = ""
+
+        for (i in 0 until distance) {
+            blocks += block
+        }
```

Comment:
`String`에 대해 호출할 수 있는 `repeat()`가 있습니다.

```suggestion
        block.repeat(distance)
```

Reply:
반영하였습니다!